sensors/shtc3Sensor/SHTC3Sensor.cpp

```
@@ -0,0 +1,146 @@
+    return (shtc1_sleep() == STATUS_OK);
+}
+
+bool SHTC3Sensor::wakeUp() {
+    return (shtc1_wake_up() == STATUS_OK);
+}
+
+SHTC3Sensor::~SHTC3Sensor() {
+    stopContinuousMeasurement();
+}
+
+void SHTC3Sensor::setMeasurementInterval(uint32_t interval_ms) {
```

Clang (14.0.6)

[warning] method 'setMeasurementInterval' can be made static

(readability-convert-member-functions-to-static)